src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -56,7 +56,7 @@ const InfoBox: UISchemaElementComponent<{
       >
         { message.title && (
         <Typography
-          component="h2"
+          component="h3"
```

Comment:
The title in InfoBox should at be an H3 at minimum because we anticipate each page to have an H2 title. Additionally this matches parity with v2.